### ISSUE TITLE
MedicalAction action field in doc could be changed to a more detailed field

### ISSUE THREAD

**Issue by belkassaby** [phenopackets/phenopacket-schema#347]
In MedicalAction, there is the "action" field which can be either of type Procedure, Treatment, RadiationTherapy or TherapeuticRegimen. Instead of it being named "action" (in a phenopacket json file that is generated with the phenopacket-tools, there is no such field/id), we could indicate that the id in the corresponding json is either "procedure", "treatment", "radiationTherapy" or "therapeuticRegimen".

**Comment by pnrobinson** [comment]
I do not think we want to make what would be a breaking change in the schema. Software should be able to make this obvious to users?

**Comment by belkassaby** [comment]
I meant that the changes could be done in the doc instead of the software side.
